bind/uniffi/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+mod credentials;
+mod dids;
+mod dsa;
+
+mod errors;
+mod in_memory_key_manager;
+
+use crate::{
+    credentials::{
+        presentation_definition::RcbPresentationDefinition,
+        verifiable_credential_11::RcbVerifiableCredential,
+    },
+    dids::{
+        did::RcbDid,
+        methods::{
+            did_dht::{rcb_did_dht_resolve, RcbDidDht},
+            did_jwk::{rcb_did_jwk_resolve, RcbDidJwk},
+            did_web::{rcb_did_web_resolve, RcbDidWeb},
+        },
+        resolution_result::RcbResolutionResult,
+    },
+    dsa::{
+        ed25519::{rcb_ed25519_generator_generate, RcbEd25519Signer, RcbEd25519Verifier},
+        RcbSigner, RcbVerifier,
+    },
+    errors::RcbError,
+    in_memory_key_manager::RcbInMemoryKeyManager,
+};
+
+use web5::apid::{
+    credentials::{
+        presentation_definition::{
+            Constraints as RcbConstraintsData, Field as RcbFieldData, Filter as RcbFilterData,
+            InputDescriptor as RcbInputDescriptorData, Optionality as RcbOptionality,
+            PresentationDefinition as RcbPresentationDefinitionData,
+        },
+        verifiable_credential_11::{
+            CredentialSubject as RcbCredentialSubjectData,
+            VerifiableCredential as RcbVerifiableCredentialData,
+        },
+    },
+    dids::{
```

Comment:
ahh I thought the top level was going to be exported as RcbDids, to clear up the namespace in Kotlin.. but.. we can handle it at the kotlin layer :hmm: